README.md

```diff
@@ -119,12 +149,12 @@ A detailed tutorial is available here:
 
 Alternatively, to deploy manually:
 
-* Clone the repo and change directory to it
-* Log in with the [Scalingo CLI](https://cli.scalingo.com/) and create an app: `scalingo create my-parse`
-* Use the [Scalingo MongoDB addon](https://scalingo.com/addons/scalingo-mongodb): `scalingo addons-add scalingo-mongodb free`
-* Setup MongoDB connection string: `scalingo env-set DATABASE_URI='$SCALINGO_MONGO_URL'`
-* By default it will use a path of /parse for the API routes. To change this, or use older client SDKs, run `scalingo env-set PARSE_MOUNT=/1`
-* Deploy it with: `git push scalingo master`
+- Clone the repo and change directory to it
```

Comment:
Same here; these changes have no benefit, but they cause this repo to have >1k line changes.